libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1274,12 +1273,12 @@ void WaveTrack::HandleClear(double t0, double t1,
 
    for (const auto &clip : mClips)
    {
-      if (clip->BeforePlayStartTime(t0) && clip->AfterPlayEndTime(t1))
+      if (clip->ExtendsPlayRegion(t0, t1))
```

Comment:
This change is not equivalent to the original.  Do you mean `ExtendsPlayRegionOnBothSides`

Reply:
That was wrong indeed. `CoversEntirePlayRegion` is what's in use now.